FrenchFilter/sections/specific.txt

```diff
@@ -2,6 +2,7 @@
 !-------------------- Specific rules --------------------!
 !--------------------------------------------------------!
 !
+rojadirecta.fr,veopartidos.net##.overlay
```

Comment:
not reproduced

Reply:
Ok let met send screenshot.

Reply:
![image](https://user-images.githubusercontent.com/76908410/108193669-ab49e080-7150-11eb-915d-70f25945fa40.png)
Link: `https://www.rojadirecta.fr/ver/espana-laliga-santander-levante-vs-atletico-de-madrid/espn-2-en-vivo-por-internet.php`

Reply:
@Alex-302